### PR TITLE
feat(frontend): Describe statement displays the primary key

### PR DESCRIPTION
**Pull request by chenzl25**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

- `describe` statement displays the primary key.

## Example

```
dev=> describe __internal_v_9_hashjoinright_1008;
    Name     |      Type
-------------+-----------------
 b_b1        | Int32
 b_b2        | Int32
 b__row_id   | Int64
 primary key | b_b1, b__row_id
```

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features).
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

If your pull request contains user-facing changes, please specify the types of the changes, and create a release note. Otherwise, please feel free to remove this section.

### Types of user-facing changes

Please keep the types that apply to your changes, and remove those that do not apply.

- SQL commands, functions, and operators

### Release note

- `describe` statement displays the primary key.

## Refer to a related PR or issue link (optional)
https://github.com/risingwavelabs/risingwave/issues/7589

### CODE REVIEW

File: src/frontend/src/handler/describe.rs [hunk 1]
```diff
@@ -35,30 +35,81 @@ pub fn handle_describe(handler_args: HandlerArgs, table_name: ObjectName) -> Res
...
         (
-            catalogs
+            column_catalogs
                 .iter()
                 .filter(|c| !c.is_hidden)
                 .map(|c| c.column_desc.clone())
                 .collect(),
+            pk_column_catalogs
+                .iter()
+                // .filter(|c| !c.is_hidden)
```
Comment (yezizp2012) [inline src/frontend/src/handler/describe.rs:89]: I'm not sure if we should expose hidden keys like row_id to user. 🤔 Prefer to ignore it.

**Review by yezizp2012 — COMMENTED** [review]
Rest LGTM.

File: src/frontend/src/handler/describe.rs (hunk 1 above)
Comment (chenzl25) [reply src/frontend/src/handler/describe.rs:89]: I will remove the comment later, but in some cases I think hidden columns can appear at the prefix of the primary key. In those cases, if we don't expose the hidden columns, users might be confused with why primary point select doesn't work.

**Review by yezizp2012 — APPROVED** [review]
LGTM!

**Pull request merged**